code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -863,7 +890,8 @@ GLOBAL_LIST_EMPTY(PDAs)
 			to_chat(user, "<span class='notice'>You slide \the [C] into \the [src].</span>")
 			inserted_item = C
 			update_icon()
-			playsound(src, 'sound/machines/button.ogg', 50, 1)
+			if (!silent)
+				playsound(src, 'sound/machines/button.ogg', 50, 1)

```

Comment:
This is a mechanical sound, making this mutable via a software toggle makes no logical sense